src/main/java/Line.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class Line {
+    private final int lineNumber;
+    private final Dots dots;
+
+    public Line(int lineNumber, DotType... types) {
+        this(lineNumber, Arrays.asList(types));
```

Comment:
```suggestion
        this(lineNumber, List.of(types));
```
위 코드도 가능한데 어떤 api를 활용해 구현하는 것이 좋을까?